src/common/utilities/FormulaEvaluation.ts

```diff
@@ -0,0 +1,641 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { sp } from "@pnp/sp";
+import { IContext } from "../Interfaces";
+import { ASTNode, ArrayLiteralNode, Token, TokenType, ValidFuncNames } from "./FormulaEvaluation.types";
+
+export class FormulaEvaluation {
+    private webUrl: string;
+    private _meEmail: string;
+    
+    constructor(context?: IContext, webUrlOverride?: string) {
+        if (context) {
+            sp.setup({ pageContext: context.pageContext });
```

Comment:
it doesn't seem you use `sp` anyhwere.
I'd suggest removing reinitialization of PnPJS here as it is not used.

Reply:
I originally used `@pnp/sp` for the `getUserImage` function in the `FormulaEvaluation` class. Doing this resulted in the `evaluateAST` and `evaluate` methods needing to be `async` in nature - which caused a fair degree of messiness when trying to render the custom header and footer in Dynamic Form in response to form updates - I did have it working, but it didn't feel very clean. I simplified my implementation, fetching a user image using the `/_layouts/15/userphoto.aspx` URL instead and forgot to remove this reference! 

Removed as suggested 👍